### ISSUE TITLE
[Feature Request]: Token access on collection level with one-click button

### ISSUE THREAD

**Issue by bpoulaindev** [usebruno/bruno#1627]
This feature already exists in Postman and would be a useful in Bruno.
Technically it can be done using pre-request scripts but this does not seem like a clean alternative.
![image](https://github.com/usebruno/bruno/assets/64689165/745ae5b3-de7c-42dd-9724-4522f0455229)

**Comment by jtourlamain** [comment]
When you crafted a token request (its just a Form post to a token endpoint) you can just set a Post Response var (ex. cc.apiname.access_token). The var will be available to all your collection requests as long as you keep Bruno running. In your request just add an Authorization header with {{cc.apiname.access_token}}. You can also set a pre request script on your collection to add the Authorization header with access_token.

**Comment by bpoulaindev** [comment]
> When you crafted a token request (its just a Form post to a token endpoint) you can just set a Post Response var (ex. cc.apiname.access_token). The var will be available to all your collection requests as long as you keep Bruno running. In your request just add an Authorization header with {{cc.apiname.access_token}}. You can also set a pre request script on your collection to add the Authorization header with access_token.

Yeah here's my point, technically you can do whatever you want using scripts but in this specific case my team would need the token to be updated every 30 minutes, hence why having a button to refresh it would be easier than re-running a request.

**Comment by anusree-bruno** [comment]
This has been implemented in the latest OAuth2 Revamp
Please upgrade to v2+
Checkout our documentation for more info on configuring at collection level https://docs.usebruno.com/auth/oauth2-experimental/collection-level-configuration

I am closing this issue. Feel free to reopen if you are still facing issue with this.
